src/menus/NavigationMenus.cpp

```diff
@@ -29,57 +29,46 @@ void NextOrPrevFrame(AudacityProject &project, bool forward)
    auto temp1 = AButton::TemporarilyAllowFocus();
    auto temp2 = ASlider::TemporarilyAllowFocus();
    auto temp3 = MeterPanel::TemporarilyAllowFocus();
-
-   auto &toolManager = ToolManager::Get( project );
-   auto botDock = toolManager.GetBotDock();
-
-
-   // Define the set of windows we rotate among.
-   static const unsigned rotationSize = 3u;
-
-   wxWindow *const begin [rotationSize] = {
+   
+   std::vector<wxWindow*> seq = {
       ProjectWindow::Get( project ).GetTopPanel(),
       &TrackPanel::Get( project ),
-      botDock,
-   };
-
-   const auto end = begin + rotationSize;
-
-   // helper functions
-   auto IndexOf = [&](wxWindow *pWindow) {
-      return std::find(begin, end, pWindow) - begin;
+      ProjectWindow::Get(project).GetEffectsWindow(),
+      ToolManager::Get( project ).GetBotDock(),
    };
 
    auto FindAncestor = [&]() {
       wxWindow *pWindow = wxWindow::FindFocus();
-      unsigned index = rotationSize;
-      while ( pWindow &&
-              (rotationSize == (index = IndexOf(pWindow) ) ) )
+      unsigned index = seq.size();
```

Comment:
This line can be removed as index is not referenced.

Reply:
Right, thanks!